### ISSUE TITLE
ePanel option as dynamic 

### ISSUE THREAD

**Issue by jamesfosker** [elegantthemes/create-divi-extension#321]
Hi, I've created custom theme option for a client site and am wondering if there is a way to add those options to dynamic content options, they are basic text options.

the option are basic company details which I would like to add dynamically to my theme builder footer.

**Comment by xxtesaxx** [comment]
As far as I'm concerned, this is currently not possible. I'm not 100% sure but maybe custom fields created via the Advanced Custom Fields plugin might be usable. Custom codeded fields however are not working

**Comment by jamesfosker** [comment]
Yeah but I referring to custom epanel options though, not metabox fields.

**Comment by xxtesaxx** [comment]
I think there is currently no way to register custom options/theme options to be used as dynamic content.
